src/features/map/components/Map/LayerToggle.jsx

```
@@ -280,12 +300,21 @@ const LayerToggle = () => {
+    };
+
     return generateLayerToggle(
       data,
       handleChange,
       handleMapLabelsChange,
       handleConstructionColorChange,
       isConstructionColorEnabled,
+      handleUseTypeColorChange,
+      isUseTypeColorEnabled,
     );
   }, [data, isConstructionColorEnabled]);
```

ESLint

[error] Compilation Skipped: Existing memoization could not be preserved

React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. The inferred dependencies did not match the manually specified dependencies, which could cause the value to change more or less frequently than expected. The inferred dependency was `setVisibility`, but the source dependencies were [data, isConstructionColorEnabled]. Inferred different dependency than source.

  283 |   }, [data?.zone?.features, setConstructionColorMap, setColorMode]);
  284 |
> 285 |   const items = useMemo(() => {
      |                         ^^^^^^^
> 286 |     const handleChange = (e) => {
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 287 |       const { value, checked } = e.target;
      …
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 318 |     );
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 319 |   }, [data, isConstructionColorEnabled]);
      | ^^^^ Could not preserve existing manual memoization
  320 |
  321 |   return (
  322 |     <Tooltip title="Toggle Layers" styles={{ body: { fontSize: 12 } }}>

(react-hooks/preserve-manual-memoization)